src/nanovg.c

```diff
@@ -2545,11 +2545,13 @@ int nvgTextBreakLines(NVGcontext* ctx, const char* string, const char* end, floa
 					breakMaxX = 0.0;
 				}
 			} else {
-				float nextWidth = iter.nextx - rowStartX;
```

Comment:
Yes. Per the comment, sub-pixel positioning brought out a bug where the advance would be inside the last glyph's max x. 

So the code would calculate bounding box based on glyph dimensions, but calculate line break based on advance, and my last character or word would not be displayed because of line break, when rendering and clipping within bounding box.